datarepo-clienttests/src/main/resources/apipayloads/dataset-simple.json

```diff
@@ -9,15 +9,15 @@
         "columns": [
           {
             "name": "VCF_File_Name",
-            "datatype": "STRING"
+            "datatype": "string"
```

Comment:
Are the datatypes case-sensitive?

Reply:
looks like it
https://github.com/go-swagger/go-swagger/issues/303